### DISCUSSION TITLE
Pass --token via env var to vercel cli

### DISCUSSION THREAD

**Discussion by tonyxiao** [vercel/community#6343] [Help] [answered]
### Summary

we have multiple vercel accounts that we work that we'd like to scope to the repo directory. Is it psosible to pass `--token` as an env var rather than command line argument to the Vercel cli?

### Steps to Reproduce

N/A

**Comment by swarnava** [comment, marked as answer]
Hey, 

You should be able to set multiple tokens in Environment variable and pass them using `vercel deploy --token=$VERCEL_TOKEN --scope=$TEAM_SLUG` , `vercel deploy --token=$VERCEL_TOKEN2 --scope=$TEAM_SLUG2` etc

**Reply by tonyxiao** [reply]
Is it possible for vercel to accept env variable addition to explicit cli argument? I'd love to be able to just run `vercel deploy` and have it do the right thing in different repos. (we use `.envrc` to link env var to directories) to source.
